### PR TITLE
Loops.md page

### PR DESCRIPTION
**Pull request by zerbers**

### CODE REVIEW

**Comment by zerbers** [comment]
Please, review.

File: learn/loops.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+Code example:
+
+```
+a = 'Hello, world!'
+```
+
+> Blockquote example
+
+It's the link to [Install section](/learn/install.md).
```
Comment (zerbers) [inline learn/loops.md:24]: Is it a correct markdown for the link?

Comment (angarsky) [reply learn/loops.md:24]: Yes, looks good.

**Review by angarsky — APPROVED** [review]
Looks good. Let's merge.

**Pull request merged**